### ISSUE TITLE
Adding a contact from a public chat auto-scrolls you to the bottom of the chat

### ISSUE THREAD

**Issue by SlizkovaGanna** [status-im/status-react#6707]
Type: **Feature request**

Summary: When user is in the middle of reading a public chat, and he decides to add someone from that chat to contacts, after returning to the chat he is auto-scrolled to the bottom of it, and has to manually scroll up to the message he was reading previously.
This happens both for desktop and mobile (where 'back' button is present)
A good user experience would be to anchor a user at the point he left at.
For example, in **Telegram**, user stays at the same point in chat history he left, in situations when he checks out user profiles; or even switches between chats when there are no new messages in the conversation.

**Expected behaviour**

When coming back from adding user to contacts, user should be anchored in the same point of chat history

**Actual behaviour**

User is auto-scrolled to the bottom of the chat and has to manually come back to the messages he needs.

**Reproduction**

1. Open Status
2. Join any public chat with users that are not in your contact list
3. Scroll up to the middle of the conversation and add any user to your contacts
4. Go back to the public chat

**Additional Information**

Status version: Develop (d2160e)
Operating System: iOS 12.0, MacOS 10.13.4

**Comment by yenda** [comment]
Will be much easier to solve on desktop once my PR is merged as you can go-to a position place in flatlists

**Comment by churik** [comment]
@SlizkovaGanna 
duplicate of https://github.com/status-im/status-react/issues/5852

**Comment by churik** [comment]
@yenda can you please assign #5852?
